wagtail_transfer/views.py

```diff
@@ -161,4 +188,21 @@ def do_import(request):
 
     importer.run()
 
-    return redirect('wagtailadmin_explore', request.POST['dest_page_id'])
+    if dest_page_id:
+        return redirect('wagtailadmin_explore', dest_page_id)
+    else:
+        return redirect('wagtailadmin_explore', Page.objects.first().id)
```

Comment:
IIRC, there's a separate URL name for exploring the root (`redirect('wagtailadmin_explore_root`)`, I think)

Reply:
Ah thanks, updated now